src/monster.h

```diff
@@ -164,6 +165,9 @@ class monster : public Creature, public JsonSerializer, public JsonDeserializer
         int calc_movecost( const tripoint &f, const tripoint &t ) const;
         int calc_climb_cost( const tripoint &f, const tripoint &t ) const;
 
+        bool is_dangerous_field( const field &fld ) const;
+        bool is_dangerous_field( const field_entry &entry ) const;
```

Comment:
Yeah, sorry. My bad. But anyways. It would make sense to move `is_dangerous_field()`s to `Creature` instead and still override them in both `monster` and `Character`.